sashimi/hardware/scanning/scanloops.py

```diff
@@ -258,6 +259,108 @@ def fill_arrays(self):
         self.wait_signal.clear()
 
 
+class TriggeredPlanarScanLoop(ScanLoop):
+    """Class for controlling the planar scanning mode, where we image only one plane and
+    do not control the piezo and vertical galvo."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        buffer_len = int(round(self.sample_rate / self.parameters.triggering.frequency))
+        self.camera_pulses = RollingBuffer(buffer_len)
+        set_impulses(
+            self.camera_pulses.buffer,
+            1,
```

Comment:
I'm abusing the `set_impulses` function here to get a camera pulses buffer for planar mode. I think we can get rid of this function and instead simply set the first item in the buffer to 5. Increasing the frame rate will be reflected in `n_samples_period()` becoming shorter, so no further modification of the camera pulses buffer is necessary.